connector/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisRecordProcessor.scala

```diff
@@ -166,15 +168,16 @@ private[kinesis] object KinesisRecordProcessor extends Logging {
             if numRetriesLeft > 1 =>
           val backOffMillis = Random.nextInt(maxBackOffMillis)
           Thread.sleep(backOffMillis)
-          logError(s"Retryable Exception:  Random backOffMillis=${backOffMillis}", e)
+          logError(log"Retryable Exception: Random " +
+            log"backOffMillis=${MDC(MAX_BETWEEN_RETRIES, backOffMillis)}", e)
```

Comment:
One last comment: Let's rename this to make it more clear.

Reply:
Done